packages/node-sqlite3/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```

Comment:
Please check this in. 

See failure on your build https://travis-ci.org/github/mochajs/mocha-examples/builds/719604449 `"can only install packages with an existing package-lock.json"`